src/main.c

```diff
@@ -1092,7 +1099,8 @@ int main(void)
                 tim1->ccr4 = tim1->ccr3 - sysclk_us(1);
                 barrier(); /* Set post-OSD timeout /then/ enable display */
                 if (config.display_2Y)
-                    display_height = 2*height;
+                    /* dont double the 2 line bottom padding */
+                    display_height = 2*height-2;
```

Comment:
All other gaps (at top, and between text rows) are doubled though. So why not this one?

Reply:
I thought the top and bottom weren’t painted into screen memory - I’ll double check.

Without this change an extra row of pixels were rendered - possibly after the painted area

Reply:
Okay I could be missing something. I'm not too precious about this, if the OSD box looks right.

Reply:
I think you’re right tho - I think I was fooled by the font and the lack of descending characters in the examples I was testing.

The junk I was seeing was probably the odd/even stuff above

Where did the font come from ?

Reply:
I think it's an old Amiga demo 8x8 font I picked up a long time ago, and also use in other projects such as Amiga Test Kit.

Reply:
Fooled I was - Callipers and a bigger screen have settled it
backed out changes and added a couple of comments in your code